### ISSUE TITLE
failed to detect object. 

### ISSUE THREAD

**Issue by LongZhai** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#396]
I only want to detect golf ball by using webcam.
after almost 50k steps, the lose is below 0.05 and min_score_thresh is changed to 0.20 in webcam.py.

`
INFO:tensorflow:Finished evaluation at 2019-11-11-15:23:03
I1111 15:23:03.355002 140212121962368 evaluation.py:275] Finished evaluation at 2019-11-11-15:23:03
INFO:tensorflow:Saving dict for global step 4869: DetectionBoxes_Precision/mAP = -1.0, DetectionBoxes_Precision/mAP (large) = -1.0, DetectionBoxes_Precision/mAP (medium) = -1.0, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = -1.0, DetectionBoxes_Precision/mAP@.75IOU = -1.0, DetectionBoxes_Recall/AR@1 = -1.0, DetectionBoxes_Recall/AR@10 = -1.0, DetectionBoxes_Recall/AR@100 = -1.0, DetectionBoxes_Recall/AR@100 (large) = -1.0, DetectionBoxes_Recall/AR@100 (medium) = -1.0, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/classification_loss = 1.5578122e-06, Loss/localization_loss = 0.0, Loss/regularization_loss = 0.048952404, Loss/total_loss = 0.048953965, global_step = 4869, learning_rate = 0.038483858, loss = 0.048953965
I1111 15:23:03.355370 140212121962368 estimator.py:2049] Saving dict for global step 4869: DetectionBoxes_Precision/mAP = -1.0, DetectionBoxes_Precision/mAP (large) = -1.0, DetectionBoxes_Precision/mAP (medium) = -1.0, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = -1.0, DetectionBoxes_Precision/mAP@.75IOU = -1.0, DetectionBoxes_Recall/AR@1 = -1.0, DetectionBoxes_Recall/AR@10 = -1.0, DetectionBoxes_Recall/AR@100 = -1.0, DetectionBoxes_Recall/AR@100 (large) = -1.0, DetectionBoxes_Recall/AR@100 (medium) = -1.0, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/classification_loss = 1.5578122e-06, Loss/localization_loss = 0.0, Loss/regularization_loss = 0.048952404, Loss/total_loss = 0.048953965, global_step = 4869, learning_rate = 0.038483858, loss = 0.048953965
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 4869: images/model.ckpt-4869
I1111 15:23:03.378846 140212121962368 estimator.py:2109] Saving 'checkpoint_path' summary for global step 4869: images/model.ckpt-4869
INFO:tensorflow:global_step/sec: 0.122549
I1111 15:27:12.525679 140212121962368 basic_session_run_hooks.py:692] global_step/sec: 0.122549
INFO:tensorflow:loss = 0.048907146, step = 4901 (816.004 sec)
I1111 15:27:12.528848 140212121962368 basic_session_run_hooks.py:260] loss = 0.048907146, step = 4901 (816.004 sec)
`

however, no object can be detected. I even change the min_score_thresh to 0.10, but still no object is detected.
if the min_score_thresh is changed to 0.10, there should be boxes everywhere. However the webcam is still normal and no object is in detection box.

I am using SSD_mobilenet_v1_ppn_shared_box_preditor_640x640_coco14
80 golf balll pictures were downloaded from google.

could you please let me know how to fix the issue.

**Comment by aidoshacks** [comment]
Can you leave a link to the images you are training with? If you are trying to detect a golf ball from far away you aren't going to have much luck since the number of pixels that the golf ball will be made up of is limited.

**Comment by LongZhai** [comment]
here is the pictures
https://drive.google.com/open?id=1QUvYRmDJGYDnZogV-xQRHKXRUB6etz0b
https://drive.google.com/open?id=1-xDgVf3IjxFl1xHzJpMxhFcYX8NNWbLB

thanks @aidoshacks
